data/src/main/java/com/teamagam/gimelgimel/data/message/repository/cache/room/dao/OutGoingMessagesDao.java

```diff
@@ -18,6 +18,6 @@
   @Query("DELETE FROM out_going_messages WHERE out_going_id = (SELECT MIN(out_going_id) FROM out_going_messages);")
   void deleteTopMessage();
 
-  @Query("DELETE FROM out_going_messages")
```

Comment:
that nuke is needed and should be used with the other nukes to clean to database (for development uses)